.github/workflows/python-package.yml

```diff
@@ -37,3 +37,26 @@ jobs:
         flags: unittests
         name: opentimelineio-codecov
         fail_ci_if_error: true
+
+  build-docs:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - run: sudo apt-get install graphviz
+    - uses: actions/checkout@v2
+      with:
+        submodules: 'recursive'
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '10'
+    - run: |
+        cd docs/data-model
+        npm install
+        npm run build
+    - uses: peaceiris/actions-gh-pages@v3
+      if: github.ref == 'refs/heads/master'
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        publish_dir: ./docs/data-model/build
+        force_orphan: true
```

Comment:
I'm not fully up on GitHub actions, would it make sense to have this be it's own YAML file in the workflows dir?